weblab/accounts/urls.py

```diff
@@ -23,3 +23,4 @@
         name='myaccount',
     ),
 ]
+app_name = 'accounts'
```

Comment:
I'm curious, as I haven't read all the docs, but what's the reason this needs to be added now?

Reply:
This is in response to the following error message 

> django.core.exceptions.ImproperlyConfigured: Specifying a namespace in include() without providing an app_name is not supported. Set the app_name attribute in the included module, or pass a 2-tuple containing the list of patterns and app_name instead

Reply:
Ah, I see. Good reason!